GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -616,11 +653,14 @@ void FramebufferManagerCommon::CopyToColorFromOverlappingFramebuffers(VirtualFra
 		}
 
 		if (src->fb_address == dst->fb_address && src->fb_stride == dst->fb_stride) {
-			// Another render target at the exact same location but gotta be a different format, otherwise
-			// it would be the same.
-			_dbg_assert_(src->fb_format != dst->fb_format);
-			// This will result in reinterpret later, if both formats are 16-bit.
-			sources.push_back(CopySource{ src, RASTER_COLOR, 0, 0 });
+			// Another render target at the exact same location but gotta be a different format or a different stride, otherwise
+			// it would be the same, and should have been detected in DoSetRenderFrameBuffer.
+			if (src->fb_format != dst->fb_format) {
+				// This will result in reinterpret later, if both formats are 16-bit.
+				sources.push_back(CopySource{ src, RASTER_COLOR, 0, 0 });
+			} else {
+				// Happens in Prince of Persia - Revelations. Ignoring.
```

Comment:
Hm, don't we check the stride and address match above?  Doesn't this mean we have two exact matches, i.e. the comment in `GetExactVFB()` is false in this game?

-[Unknown]

Reply:
It's true that the GetExactVFB comment is not 100% accurate. This happened in POP when it ended up creating a 480x272x16 target with stride 200 at the same location as it just had a 480x272x32 target with the same stride. Curiously this went away with the FlushBeforeCopy change in the same PR, I think it temporarily sets a 16-bit resolution without drawing anything with it, due to some legacy code or something.

I realize I do need to take account for this situation properly.